### PR TITLE
Add new issue template for a general feature

### PR DESCRIPTION
**Pull request by IT-VBFK**
This is meant for changes, that should be discussed first, but neither fit into the `bug` nor in the `api-proposal` template/category.

This looks like this:

## IMPORTANT 

* [ ] If the PR touches the public API, the changes have been approved in a separate issue with the "api-approved" label.
* [ ] The code complies with the [Coding Guidelines for C#](https://www.csharpcodingguidelines.com/).
* [ ] The changes are covered by unit tests which follow the Arrange-Act-Assert syntax and the naming conventions such as is used [in these tests](../tree/develop/Tests/FluentAssertions.Equivalency.Specs/MemberMatchingSpecs.cs#L51-L430).
* [ ] If the PR adds a feature or fixes a bug, please update [the release notes](../tree/develop/docs/_pages/releases.md) with a functional description that explains what the change means to consumers of this library, which are published on the [website](https://fluentassertions.com/releases).
* [ ] If the PR changes the public API the changes needs to be included by running [AcceptApiChanges.ps1](../tree/develop/AcceptApiChanges.ps1) or [AcceptApiChanges.sh](../tree/develop/AcceptApiChanges.sh).
* [ ] If the PR affects [the documentation](../tree/develop/docs/_pages), please include your changes in this pull request so the documentation will appear on the [website](https://www.fluentassertions.com/introduction).
    * [ ] Please also run `./build.sh --target spellcheck` or `.\build.ps1 --target spellcheck` before pushing and check the good outcome

### CODE REVIEW

**Review by dennisdoomen — APPROVED** [review]
Can you add a screenshot of what that will look like?

**Comment by IT-VBFK** [comment]
Sure :)

image

I've kept it as simple as possible, because a feature not regarding the public api surface does not need certain fields

File: .github/ISSUE_TEMPLATE/03_general_feature.yml
Comment (jnyrup) [inline .github/ISSUE_TEMPLATE/03_general_feature.yml:9]: Not sure if I'm fond of the term "feature request", maybe it's just me, but it has some demanding tone in my ears.

How about this?
```suggestion
        You have an idea for a new feature? Cool, let's discuss it.
```

File: .github/ISSUE_TEMPLATE/03_general_feature.yml
Comment (jnyrup) [inline .github/ISSUE_TEMPLATE/03_general_feature.yml:2]: PRs add new features, issues don't.

```suggestion
description: Discuss a new feature that does not change the public API surface
```

Comment (IT-VBFK) [reply .github/ISSUE_TEMPLATE/03_general_feature.yml:2]: 😆 

yepp :)

**Review by jnyrup — APPROVED** [review]

**Comment by IT-VBFK** [comment]
@jnyrup Do we miss anything in the template? I was not sure, if a general feature should have a proposal field, were one can add a little snippet.

File: .github/ISSUE_TEMPLATE/03_general_feature.yml
Comment (IT-VBFK) [reply .github/ISSUE_TEMPLATE/03_general_feature.yml]: Do you mean, that one can think: "Oh.. let's request something and the maintainers have to deliver"?

Comment (jnyrup) [reply .github/ISSUE_TEMPLATE/03_general_feature.yml]: Exactly.
I had to lookup "request" in a dictionary to double check the meaning of it.

I'm not strictly against, just wanted bring it up before merging.

Comment (IT-VBFK) [reply .github/ISSUE_TEMPLATE/03_general_feature.yml]: I appreciate the effort of reviewing 👍🏻

Comment (IT-VBFK) [reply .github/ISSUE_TEMPLATE/03_general_feature.yml]: > I had to lookup "request" in a dictionary to double check the meaning of it.

If you take google translate: for the word "request" there are more synonyms for "come on do that.. now" than "please can you.."

So.. I am fine with this formulation

**Pull request merged**
